guides/common/attributes.adoc

```diff
@@ -284,3 +284,104 @@ ifdef::katello[]
+:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-beta-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-6-beta-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.7-rpms
+:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
+:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.8-rpms
+:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
+:SatelliteAnsibleVersion: 2.9
+:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+:SmartProxies: orcharhino Proxies
+:SmartProxy: orcharhino Proxy
+:SmartProxyServer: orcharhino Proxy
+:SpecialCaseProductVersion: 5.5
+:TargetVersion: 5.5
+:TargetVersionMaintainUpgrade: 5.6
+:Team: ATIX AG
+:ansible-doc-activation_key: ansible-doc theforeman.foreman.activation_key
+:ansible-galaxy: https://galaxy.ansible.com/theforeman/foreman
+:ansible-namespace-example: "`theforeman.foreman._module_name_`"
+:ansible-namespace: "`theforeman.foreman`"
+:ansiblefilepath: /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+:awx: AWX
+:certs-generate: foreman-proxy-certs-generate
+:certs-proxy-context: foreman-proxy
+:customcontent: custom content
+:customcontenttitle: Custom Content
+:customfiletype: custom file type
+:customfiletypetitle: Custom File Type
+:customgpgtitle: Custom GPG
+:customostreecontent: custom OSTree content
+:customostreecontenttitle: Custom OSTree Content
+:customproduct: custom product
+:customproducttitle: Custom Product
+:customrepo: custom repository
+:customrpm: custom RPM
+:customrpmtitle: Custom RPM
+:customssl: custom SSL
+:customssltitle: Custom SSL
+:foreman-example-com: orcharhino.example.com
+:foreman-installer: foreman-installer
+:foreman-maintain: foreman-maintain
+:installer-scenario-smartproxy: orcharhino-installer --no-enable-foreman
+:installer-scenario: foreman-installer --scenario katello
+:oVirt: oVirt
+:oVirtEngine: oVirt Engine
+:oVirtShort: oVirt
+:ovirt-example-com: ovirt.example.com
+:package-clean: yum clean
+:package-install-project: yum install
+:package-install: yum install
+:package-remove-project: yum remove
+:package-remove: yum remove
+:package-update-project: yum update
+:package-update: yum update
+:project-change-hostname: katello-change-hostname
+:project-client-RHEL7-url: https://yum.theforeman.org/client/{2.1.3}/el7/x86_64/foreman-client-release.rpm
+:project-client-name: orcharhino client
+:project-context: orcharhino
+:project-installation-guide-title: Installing orcharhino
+:provision-script: OS installer recipe
+:smart-proxy-context: orcharhino Proxy
+:smart-proxy-installation-guide-title: Installing orcharhino Proxy
+:smartproxy-example-com: orcharhino-proxy.example.com
+:smartproxy_port: 8443
+endif::[]
```

Comment:
My only concern here is now that we have four build types, the list is getting a bit out of control. Only Satellite appears to name thing differently, foreman-el, foreman-deb

Reply:
I am working on a patch that should help to clean up the mess, if you woudln't mind dropping your opinion there and then after hopefully it is merged rebasing on top of that.